back-end/requests/put/add_members.js

```diff
@@ -0,0 +1,43 @@
+let Playlist = require("../../models/playlists.model");
+
+const add_member = async (req, res) => {
+  const group_id = req.params.group_id
+  const user_id = req.params.user_id
+
+  let members
+  let banned_members
+  await Playlist.findOne({_id:group_id}) //retrieve the group information
+    .then(response => {
+      members = response.members
+      banned_members = response.banned_members
+    })
+    .catch(err => {
+      const msg = "Cannot find the group" 
+      console.log(msg)
+      return new Error(msg)
+    })
```

Comment:
We should now check if an error was returned by the promise from above. So here we should have a check, which passes the error to middleware if it exists.
```
if (error)
{
  return next(error)
}
```